aerialist/ros_node_script/spawn_script/spawn_pattern_box.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+import rospy
+import os
+import sys
+from gazebo_msgs.srv import SpawnModel
+from geometry_msgs.msg import Pose
+
+
+def spawn_box():
+    print(len(sys.argv))
+    print(sys.argv)
+    box_name = sys.argv[1]
+    l = int(sys.argv[2])
+    w = int(sys.argv[3])
+    h = int(sys.argv[4])
+    x = int(sys.argv[5])
+    y = int(sys.argv[6])
+    z = int(sys.argv[7])
+    # Initialize the ROS node
+    rospy.init_node(box_name)
+    # rospy.spin()
+
+    # Wait for the Gazebo spawn service
+    rospy.wait_for_service("/gazebo/spawn_sdf_model")
+    spawn_sdf_model = rospy.ServiceProxy("/gazebo/spawn_sdf_model", SpawnModel)
+
+    # Define the SDF model for a box with the specified size
+    box_sdf = """
```

Comment:
why not put this texts (also for other obstacles) in a .sdf file instead of hardcoded here?
let's discuss it